github/rules.go

```diff
@@ -217,6 +218,19 @@ type RepositoryRulesetRefConditionParameters struct {
 	Exclude []string `json:"exclude"`
 }
 
+// RepositoryRulesetOrganizationPropertyTargetParameters represents an organization_property name and values to be used for targeting.
+type RepositoryRulesetOrganizationPropertyTargetParameters struct {
```

Comment:
We already have the `RepositoryRulesetRepositoryPropertyTargetParameters` type, we should re-use this type.